geomesa-convert/geomesa-convert-all/src/test/scala/org/locationtech/geomesa/convert/URLConfigProviderTest.scala

```diff
@@ -32,41 +32,73 @@ class URLConfigProviderTest extends Specification {
           req.asInstanceOf[Request].setHandled(true)
           resp.getWriter.write(
             """
-              |geomesa = {
-              |  sfts = [
-              |    {
-              |      type-name = "example-csv-fromurl"
+              |geomesa {
+              |  sfts {
+              |    "example-csv-url" = {
```

Comment:
So, ```geomesa = {``` changed to ```geomesa {```, but ```"example-csv-url" = {``` remained...  I've read the HOCON syntax, but didn't see a difference for this.  What did I miss?